### ISSUE TITLE
Copying files from CD failed for discs in languages other than English

### ISSUE THREAD

**Issue by ClarkSteel** [OpenRA/OpenRA#11668]
openra not find the CD, he gives me "error: copyng files from cd failed" ... I think it's because I have an ISO image of the CD in my language, regularly mounted, but he tries in the physical unit. How can I make him read other units?

**Comment by Mailaender** [comment]
https://github.com/OpenRA/OpenRA/blob/bleed/mods/ra/mod.yaml#L229 liststhe currently supported discs. You will need to expand those entries. The metadata can be read with the following `OpenRA.Utility.exe` formats.

```
  --list-installshield ARCHIVE.Z
  Lists the content ranges for a InstallShield V3 file

  --list-installshield-cab DATA.HDR
  Lists the filenames contained within an Installshield CAB volume set

  --list-mix ARCHIVE.MIX
  Lists the content ranges for a mix file

  --list-mscab ARCHIVE.CAB
  Lists the filenames contained within a MSCAB file
```

If you have no idea what I am talking about and want to help fill the gaps, you should join the chat room, see http://www.openra.net/community/

**Comment by pchote** [comment]
This should be "fixed" in playtest-20160904 by explicitly requiring the english CDs, instead of accepting any disk and then failing.

**Comment by Mailaender** [comment]
https://github.com/OpenRA/OpenRA/tree/bleed/mods/ra/installer is now the location for CD layout metadata, but I think I also misunderstood what the feature request is about. I guess this is about mounted CD images which is a duplicate of https://github.com/OpenRA/OpenRA/issues/5970 and should just work now if it is really mounted. Be sure to open it up in the file browser.
